README.md

```diff
@@ -52,7 +52,7 @@ Run the node using one of these commands:
 ```sh
 # Replace host and port with appropriate values
 ros2 run hlvs_player hlvs_player --ros-args -p host:="127.0.0.1" -p port:=10001
-ros2 launch hlvs_player hlvs_player.launch
+ros2 launch hlvs_player example.launch
```

Comment:
There's one more reference to ``hlvs_player.launch`` on line 15 in this file which needs to be updated.